### ISSUE TITLE
E_DEPRECATED notice in Zend_Mail

### ISSUE THREAD

**Issue by shmaltorhbooks** [zendframework/zf1#555]
php version: PHP 5.6.7

code
```php
$mail = new \Zend_Mail();
// ...
$mail->send(new \Zend_Mail_Transport_Smtp($a, $b));
```
generates E_DEPRECATED notice in file Zend/Validate/Hostname.php on line 649 with message:
> iconv_set_encoding(): Use of iconv.internal_encoding is deprecated

**Comment by mhujer** [comment]
Are you using latest version of ZF1? I believe it has been fixed in some of the recent releases.

**Comment by froschdesign** [comment]
This was fixed with version 1.12.7!
